docgen/src/guides/no-result.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Designing the no-result page
+mainTitle: Guides
+layout: main.pug
+category: guides
+withHeadings: true
+navWeight: 10
+editable: true
+githubSource: docgen/src/guides/no-result.md
+---
+## Designing the no results page
+
+When the query returns no results, it is important to let the user know that their query led to no results.
+
+By doing so, we’re acknowledging that not all queries lead to some result and with some additional hints we can let them continue to use the application. This way, there is less chance that the user will leave the website to do a search to an external search engine.
+
+There are a few strategies that can be implemented from the very simple to some more complex strategies. This guide will walk you through one that can be easily implemented with InstantSearch.js: we’ll first improve the message that we provide to the user and then we’ll add a button to let the user clear the filters.
+
+## Display a message
+
+By default, InstantSearch.js will only display "no results" when there are no results. The bare minimum to handle the no-result case is to provide the user with a message that indicates that no results were found in a friendly fashion.
+
+In order to do that, we can use the InstantSeach.js’ Hits widget option [templates.empty](https://community.algolia.com/instantsearch.js/v2/widgets/hits.html#struct-HitsTemplates-empty):
+
+```javascript
+const search = instantsearch(/* parameters */);
+search.addWidget(instantsearch.widgets.hits({
+  container: ‘hits’,
+  templates: {
+    empty: ‘<p class="info”>No results were found with your current filters. Try to remove some filters or change the search query.</p>’,
```

Comment:
It should be `<p class="info">` and not this weird `”` character from google docs.